src/capymoa/regressor/_soknl_base_tree.py

```diff
@@ -11,7 +11,29 @@
 
 
 class SOKNLBT(MOARegressor):
-    """The base tree for Self-Optimising K Nearest Leaves as distribed by Sun. at el."""
+    """The base tree for Self-Optimising K Nearest Leaves as distribed by Sun. at el.
+
+    SOKNLBT is a modification to the FIMT-DD that allows FIMT-DD to store necessary information.
```

Comment:
SOKNLBT modifies the FIMT-DD algorithm to store information at the leaves that allows SOKNL to calculate the distance between a given instance and a leaf. 

See also... (add link to SOKNL) and (add link to FIMTDD)